environment.yml

```diff
@@ -20,4 +20,5 @@ dependencies:
       - pytest-ordering
       - git+git://github.com/cta-observatory/ctapipe_io_lst@v0.6.5
       - ctaplot
+      - astropy>=4.1
```

Comment:
current `setup.py` requires `astropy~=4.2`. Should it be the same here, just to be consistent?

Reply:
yes.

Reply:
This entry is now duplicated in the environment. It appears once in the conda packages and once in the pip section with different versions. Please just remove this line.

Reply:
Ok